Clean docs and update readme (#9)
bcgov/bcstats-ses: README.md

<!-- 
Add a project state badge
<!-- badges: start -->
[![Lifecycle:Experimental](https://img.shields.io/badge/Lifecycle-Experimental-339999)](https://github.com/bcgov/repomountie/blob/master/doc/lifecycle-badges.md)
[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/license/apache-2-0/)
<!-- badges: end -->

BC community Socio-economic status index
============================

> BC community socio-economic status (SES) index evaluates individuals or groups' relative social and economic standing within a BC.

Reviewer: ```suggestion
The BC community socio-economic status (SES) index evaluates individuals or groups' relative social and economic standing within BC.
```

> The SES index is often used in research, public policy, and social programs to understand disparities in outcomes such as health, education, and employment, and to develop interventions aimed at reducing inequality and promoting social mobility.

Reviewer: I would add a line along the lines of...

"This repo is utilized to clean externally available sources of data such that it can be ingested for further use within a secure research environment."

Author: I have added it.

### Data

Data used in this project is stored on the LAN. See `src/README.md` for detail.

### Usage

> The code for creating a SES are including in the `src`. 

Reviewer: ```suggestion
The code for cleaning external data to be used in the development of a SES index are including in the `src` folder. More details for running this code can be found in /src/README.md.  
```

> A R env environment is created for this project using [`renv` package](https://rstudio.github.io/renv/articles/renv.html). You can download and install all the packages this project needs by running `renv::restore()`.

Reviewer: ```suggestion
An R env environment is created for this project using [`renv` package](https://rstudio.github.io/renv/articles/renv.html). You can download and install all the packages this project needs by running `renv::restore()`.
```
